help_to_heat/frontdoor/views.py

```diff
@@ -137,6 +137,27 @@ def get_prev_next_urls(session_id, page_name):
     return prev_page_url, next_page_url
 
 
+def get_supplier_and_add_comma_after_bulb(session_id):
```

Comment:
Inclined to say we should wrap these three methods up in a class (probably one that takes `session_id` in the constructor). Maybe `BulbSupplierConverter`?

No real functional change, but just grouping them together.

Might even be worth moving that class to `interface`, including the current `sanitise_data_for_referral` logic as `replace_bulb_with_octopus_in_data` or something, and then `sanitise` just becomes `data = replace_bulb_with_octopus_in_data(data); return data`. Just since it's hanging on the same logic as the other three.

None of this is essential, what's here is fine, but would be good to keep related stuff grouped more explicitly 🙂

Reply:
Cool, let me give it a try :)